src/course-unit/add-component/AddComponent.jsx

```diff
@@ -122,6 +137,18 @@ const AddComponent = ({ blockId, handleCreateNewCourseXBlock }) => {
           );
         })}
       </ul>
+      <StandardModal
+        title="Select component"
+        isOpen={isAddLibraryContentModalOpen}
+        onClose={closeAddLibraryContentModal}
+        isOverflowVisible={false}
+        size="xl"
+      >
+        <ComponentPicker
+          showOnlyPublished
+          onComponentSelected={handleLibraryV2Selection}
+        />
+      </StandardModal>
```

Comment:
We have the `ComponentPickerModal`, but I'm fine as you did here.
Maybe we shouldn't need the modal component after all.

Reply:
@rpenido Yes, we can use it but then have to modify it to have the same props as the `ComponentPicker` component in addition to `StandardModal` props.

> Maybe we shouldn't need the modal component after all.

Agreed, it doesn't seem to be doing much except for wrapping component picker in a modal component.